_includes/nav.html

```diff
@@ -1,57 +1,29 @@
+      <span></span>
+      <span></span>
+      <span></span>
     </div>
+  </div>
 
-    <div id="navbarExampleTransparentExample" class="navbar-menu">
-        <div class="navbar-start">
-            <a class="navbar-item" href="/blogs">
-                Activities
-            </a>
-            <a class="navbar-item" href="/projects">
-                Projects
-            </a>
-            <a class="navbar-item" href="/blogs">
-                Blogs
-            </a>
-            <a class="navbar-item" href="/blogs">
-                Archives
-            </a>
-        </div>
+  <div id="navbarExampleTransparentExample" class="navbar-menu is-active">
+    <div class="navbar-start">
+      <a class="navbar-item" href="/category/activities">Activities</a>
+      <a class="navbar-item" href="/projects">Projects</a>
+      <a class="navbar-item" href="/blog">Blogs</a>
+      <a class="navbar-item" href="/archive">Archives</a>
+    </div>
 
-        <div class="navbar-end">
-            <div class="navbar-item">
-                <div class="field is-grouped">
-                    <p class="control">
-                        <a class="bd-tw-button" data-social-network="Twitter" data-social-action="tweet" data-social-target="http://localhost:4000" target="_blank" href="https://twitter.com/intent/tweet?text=Bulma: a modern CSS framework based on Flexbox&amp;hashtags=bulmaio&amp;url=http://localhost:4000&amp;via=jgthms">
-                          <span class="icon">
-                            <i class="fab fa-twitter"></i>
-                          </span>
-                        </a>
-                    </p>
-                    <p class="control">
-                        <a class="bd-tw-button" data-social-network="Twitter" data-social-action="tweet" data-social-target="http://localhost:4000" target="_blank" href="https://twitter.com/intent/tweet?text=Bulma: a modern CSS framework based on Flexbox&amp;hashtags=bulmaio&amp;url=http://localhost:4000&amp;via=jgthms">
-                          <span class="icon">
-                            <i class="fab fa-github"></i>
-                          </span>
-                        </a>
-                    </p>
-                    <p class="control">
-                        <a class="bd-tw-button" data-social-network="Twitter" data-social-action="tweet" data-social-target="http://localhost:4000" target="_blank" href="https://twitter.com/intent/tweet?text=Bulma: a modern CSS framework based on Flexbox&amp;hashtags=bulmaio&amp;url=http://localhost:4000&amp;via=jgthms">
-                          <span class="icon">
-                            <i class="fas fa-rss"></i>
-                          </span>
-                        </a>
-                    </p>
-                </div>
-            </div>
+    <div class="navbar-end">
+      <div class="navbar-item">
+        <div class="field is-grouped">
+          {% include social-links.html %}
```

Comment:
Remove social-links from navbar now that social links are moved to footer

Reply:
Done, But right side of navbar no content now ! 
@zoras  dai, Can we add the rubynepal's email there ?

Reply:
ya good point, add `info [at] rubynepal d0t org`